ui/app/adapters/ldap/role.js

```diff
@@ -66,8 +66,9 @@ export default class LdapRoleAdapter extends NamedPathAdapter {
         this.flashMessages.info(`Error fetching roles from ${errorMessages.join(', ')}`);
       }
     }
-
-    return roles.sortBy('name');
```

Comment:
thank you for the comment.

Reply:
So weird. If I just returned an array or returned the roles under the data key it wouldn't work. I started looking into it but couldn't easily find why. It runs through a few application serializer methods (normalizeItems and normalizeResponse) and it must be something in there.